### ISSUE TITLE
FontForge not found while uploading custom font

### ISSUE THREAD

**Issue by Torvaldi** [penpot/penpot#1036]
Hello, I'm trying to upload a custom font to my self-hosted penpot instance but it look's like that something is missing in the official Docker image...

**Describe the bug**
Uploading custom font returns a "Cannot run program \"fontforge\": error=2, No such file or directory"

**To Reproduce**
1.  Go to Fonts
2.  Click on Add Custom Font
3.  Click upload
4.  See error in network's tab in console

**Expected behavior**
The font should be uploaded correctly

**Screenshots**
![image](https://user-images.githubusercontent.com/7703951/122037948-63bd2380-cdd5-11eb-9244-81ca7dfb8f8e.png)

**Desktop (please complete the following information):**

-   OS: Ubuntu 20.04
-   Browser: Chrome 91.0.4472.101
-   Penpot Version: Alpha version 1.6.4

**Environment (please complete the following information):**
Self-hosted instance on Docker

-   OS Version: Ubuntu 20.04.2 LTS
- Docker version: 20.10.5, build 55c4c88
- Docker-compose version: 1.28.5, build c4eb3a1f

Also provide Docker commands or docker-compose file if possible and if proceed.x

-   Docker / Docker-compose Version: (e.g. Docker version 18.03.0-ce, build 0520e24)
-   Image (e.g. alpine)

**Frontend Stack Trace (if self-hosted)**

<details>

```
172.18.0.7 - - [15/Jun/2021:10:34:46 +0000] "POST /api/rpc/mutation/create-font-variant HTTP/1.1" 500 145 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.101 Safari/537.36"
```

</details>

**Backend Stack Trace (if self-hosted)**

<details>

```
[2021-06-15 10:34:46.479] [qtp856622266-63] E app.http.errors - error-id="4b15bfa0-cdc5-11eb-b486-ac70f0d9255f" error-message="Cannot run program "fontforge": error=2, No such file or directory" hint="internal error"
java.io.IOException: Cannot run program "fontforge": error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1142) ~[?:?]
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1073) ~[?:?]
	at java.lang.Runtime.exec(Runtime.java:590) ~[?:?]
	at clojure.java.shell$sh.invokeStatic(shell.clj:113) ~[clojure-1.10.3.jar:?]
	at clojure.java.shell$sh.doInvoke(shell.clj:79) ~[clojure-1.10.3.jar:?]
	at clojure.lang.RestFn.invoke(RestFn.java:457) ~[clojure-1.10.3.jar:?]
	at app.media$eval35641$fn__35643$ttf__GT_otf__35645.invoke(media.clj:229) ~[?:?]
	at app.media$eval35641$fn__35643$fn__35670.invoke(media.clj:302) ~[?:?]
	at app.media$eval35641$fn__35643$gen_if_nil__35668.invoke(media.clj:294) ~[?:?]
	at clojure.core$update.invokeStatic(core.clj:6187) ~[clojure-1.10.3.jar:?]
	at clojure.core$update.invoke(core.clj:6177) ~[clojure-1.10.3.jar:?]
	at app.media$eval35641$fn__35643.invoke(media.clj:302) ~[?:?]
	at clojure.lang.MultiFn.invoke(MultiFn.java:229) ~[clojure-1.10.3.jar:?]
	at app.media$run.invokeStatic(media.clj:80) ~[?:?]
	at app.media$run.invoke(media.clj:71) ~[?:?]
	at app.rpc.mutations.fonts$create_font_variant.invokeStatic(fonts.clj:49) ~[?:?]
	at app.rpc.mutations.fonts$create_font_variant.invoke(fonts.clj:47) ~[?:?]
	at app.rpc.mutations.fonts$sm$create_font_variant$fn__38238.invoke(fonts.clj:45) ~[?:?]
	at next.jdbc.transaction$transact_STAR_.invokeStatic(transaction.clj:72) ~[?:?]
	at next.jdbc.transaction$transact_STAR_.invoke(transaction.clj:51) ~[?:?]
	at next.jdbc.transaction$eval6933$fn__6934.invoke(transaction.clj:136) ~[?:?]
	at next.jdbc.protocols$eval5732$fn__5733$G__5723__5742.invoke(protocols.clj:57) ~[?:?]
	at next.jdbc$transact.invokeStatic(jdbc.clj:342) ~[?:?]
	at next.jdbc$transact.invoke(jdbc.clj:334) ~[?:?]
	at app.rpc.mutations.fonts$sm$create_font_variant.invokeStatic(fonts.clj:42) ~[?:?]
	at app.rpc.mutations.fonts$sm$create_font_variant.invoke(fonts.clj:40) ~[?:?]
	at app.metrics$wrap_summary$fn__4227.invoke(metrics.clj:264) ~[?:?]
	at clojure.lang.AFn.applyToHelper(AFn.java:156) ~[clojure-1.10.3.jar:?]
	at clojure.lang.RestFn.applyTo(RestFn.java:132) ~[clojure-1.10.3.jar:?]
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31) ~[clojure-1.10.3.jar:?]
	at clojure.lang.RestFn.invoke(RestFn.java:421) ~[clojure-1.10.3.jar:?]
	at app.rpc$wrap_impl$fn__28152.invoke(rpc.clj:103) ~[?:?]
	at app.rpc$rpc_mutation_handler.invokeStatic(rpc.clj:59) ~[?:?]
	at app.rpc$rpc_mutation_handler.invoke(rpc.clj:50) ~[?:?]
	at app.rpc$eval28198$fn__28199$fn__28202.invoke(rpc.clj:183) ~[?:?]
	at app.http.session$middleware$fn__26345.invoke(session.clj:69) ~[?:?]
	at ring.middleware.cookies$wrap_cookies$fn__28950.invoke(cookies.clj:214) ~[?:?]
	at app.http.middleware$wrap_errors$fn__29185.invoke(middleware.clj:109) ~[?:?]
	at app.http.middleware$wrap_parse_request_body$fn__29175.invoke(middleware.clj:56) ~[?:?]
	at ring.middleware.keyword_params$wrap_keyword_params$fn__28996.invoke(keyword_params.clj:53) ~[?:?]
	at ring.middleware.multipart_params$wrap_multipart_params$fn__29128.invoke(multipart_params.clj:171) ~[?:?]
	at ring.middleware.params$wrap_params$fn__29152.invoke(params.clj:67) ~[?:?]
	at app.http.middleware$wrap_format_response_body$fn__29181.invoke(middleware.clj:97) ~[?:?]
	at app.http.middleware$wrap_etag$fn__29199.invoke(middleware.clj:149) ~[?:?]
	at reitit.ring$ring_handler$fn__31016.invoke(ring.cljc:329) ~[?:?]
	at app.http.middleware$wrap_server_timing$fn__29162.invoke(middleware.clj:26) ~[?:?]
	at clojure.lang.AFn.applyToHelper(AFn.java:154) ~[clojure-1.10.3.jar:?]
	at clojure.lang.AFn.applyTo(AFn.java:144) ~[clojure-1.10.3.jar:?]
	at clojure.lang.AFunction$1.doInvoke(AFunction.java:31) ~[clojure-1.10.3.jar:?]
	at clojure.lang.RestFn.invoke(RestFn.java:408) ~[clojure-1.10.3.jar:?]
	at app.http$router_handler$fn__31660.invoke(http.clj:90) ~[?:?]
	at ring.adapter.jetty9$proxy_handler$fn__31561.invoke(jetty9.clj:54) ~[?:?]
	at ring.adapter.jetty9.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle(Unknown Source) ~[?:?]
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:51) ~[jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:171) ~[jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122) ~[jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.server.Server.handle(Server.java:562) ~[jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$0(HttpChannel.java:399) ~[jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:656) [jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:391) [jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:284) [jetty-server-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:319) [jetty-io-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100) [jetty-io-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.io.SocketChannelEndPoint$1.run(SocketChannelEndPoint.java:101) [jetty-io-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333) [jetty-util-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310) [jetty-util-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168) [jetty-util-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126) [jetty-util-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:378) [jetty-util-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:894) [jetty-util-10.0.2.jar:10.0.2]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1038) [jetty-util-10.0.2.jar:10.0.2]
	at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.ProcessImpl.forkAndExec(Native Method) ~[?:?]
	at java.lang.ProcessImpl.<init>(ProcessImpl.java:313) ~[?:?]
	at java.lang.ProcessImpl.start(ProcessImpl.java:244) ~[?:?]
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1109) ~[?:?]
	... 71 more

```

</details>

**Comment by niwinz** [comment]
Clearly a bug, thanks for reporting.

**Comment by niwinz** [comment]
Should be fixed in 1.6.5-alpha image

**Comment by Torvaldi** [comment]
> Should be fixed in 1.6.5-alpha image

Yes fixed, thank you very much!
